{"name": "test_landslide_case_start", "status": "failed", "statusDetails": {"message": "AssertionError: landslide is running a case!\nassert False", "trace": "get_test_id = False\n\n    def test_landslide_case_start(get_test_id):\n>       assert get_test_id, \"landslide is running a case!\"\nE       AssertionError: landslide is running a case!\nE       assert False\n\ntest_001_pdu_establishment.py:48: AssertionError"}, "attachments": [{"name": "log", "source": "87b9d7ec-c867-4ac3-a1b0-9b19e691adac-attachment.txt", "type": "text/plain"}], "start": 1566874060434, "stop": 1566874060435, "uuid": "414892e2-578f-4f8f-a184-ca3f5a67e998", "historyId": "9eb2b9f80043f88b2f58d008f2727442", "testCaseId": "b3f4749ff5dc127898a5c843642053f0", "fullName": "test_001_pdu_establishment#test_landslide_case_start", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_001_pdu_establishment"}, {"name": "subSuite"}, {"name": "host", "value": "ubuntu18"}, {"name": "thread", "value": "17389-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_001_pdu_establishment"}]}